src/lib/ColumnToCode.php

```diff
@@ -227,14 +273,37 @@ private function resolve():void
                 $this->column->dbType . (strpos($this->column->dbType, '(') !== false ? '' : $rawSize);
         } elseif ($this->isEnum()) {
             $this->resolveEnumType();
+        } elseif ($this->isDecimal()) {
+            $this->fluentParts['type'] = $dbType;
+            $this->rawParts['type'] = $dbType;
         } else {
             $this->fluentParts['type'] = $type . $fluentSize;
             $this->rawParts['type'] =
                 $this->column->dbType . (strpos($this->column->dbType, '(') !== false ? '' : $rawSize);
         }
+        
+        $this->isBuiltinType = $this->getIsBuiltinType($type, $dbType);
+
         $this->resolveDefaultValue();
     }
 
+    /**
```

Comment:
please add a description what "builtin type" means.
also add a comment that explains why this returns false for the combination of enum and mariadb.

Reply:
I have added docs for what builtInType means. But for the return false part for enum and Mariadb, I am not able to figure out completely. Possibly enum does not have its corresponding method in \yii\db\SchemaBuilderTrait. And I think for all DB enum should return `false`. Also this is added by @siggi-k , at this moment I don't have concrete reason. I will look at this again when I take issue #111 in hand